resource_management_mobile/src/app/pages/resource-requirement/add-resource-requirement/add-resource-requirement.component.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../../../theme/theme.scss";
```

Comment:
Don't use theme.scss everywhere use it globally or declare in angular.json

Reply:
done